deploy.sh

```diff
@@ -408,10 +410,6 @@ function openshift_deploy() {
 }
 
 function openshift_start() {
-	echo
-	echo "OpenShift support coming soon!"
-	exit -1;
-
```

Comment:
Need to be consistent for all installs
```
    echo
    echo "###   Installing kruize for OpenShift"
    echo
```

Reply:
Need to add the warning as above and ask for confirmation from user to proceed. For automated mode, warning still needs to be printed but no need for confirmation.

Reply:
Should the warning message and the confirmation be in function `openshift_start` or `openshift_prereq` (like minikube)?

Reply:
It is good to do it in `_start`, that way no damage done if the user chooses 'no'.

Reply:
Added warning and confirmation